examples/dgmulti_1d/elixir_euler_flux_diff.jl

```diff
@@ -0,0 +1,34 @@
+using Trixi, OrdinaryDiffEq
+
+dg = DGMulti(polydeg = 3, element_type = Line(), approximation_type = Polynomial(),
+             surface_integral = SurfaceIntegralWeakForm(FluxLaxFriedrichs()),
```

Comment:
You also have the keyword argument `surface_flux` that would insert the flux into a `SurfaceIntegralWeakForm`, don't you? This keyword would be more consistent with other elixirs we have (using `DGSEM`).